contracts/common/GetActionSummary.sol

```diff
@@ -147,17 +144,16 @@ contract GetActionSummary is ExtractCallData, GetActionDomainSkillId {
         totalSummary.domainSkillId = actionSummary.domainSkillId;
       }
 
-      if (isSpecialFunction(actionSummary.sig)) {
-        // If any of the actions are NO_ACTION or OLD_MOVE_FUNDS,
+      if (sigInArray(actionSummary.sig, _dominantSigs)) {
+        // If any of the actions are classed as 'dominant',
         //   the entire multicall is such
-        totalSummary.sig = actionSummary.sig;
+        return actionSummary;
+      } else if (sigInArray(actionSummary.sig, _forbiddenSigs)) {
+        revert("colony-action-summary-forbidden-sig");
```

Comment:
From the perspective of logical flow, I would flip these around such that the forbidden check comes first.